python/cudf/cudf/__init__.py

```diff
@@ -113,8 +112,7 @@
 rmm.register_reinitialize_hook(clear_cache)
 
 
-__version__ = get_versions()["version"]
-del get_versions
+__version__ = "23.04.00"
```

Comment:
Is this actually the right version for a pre-release of `23.04`?

I have:

```python
from packaging import version
version.parse("23.04.00") < version.parse("23.04")
# => False
version.parse("23.04.00") == version.parse("23.04")
# => True
```

I would have thought we want something like `23.04.a`

Reply:
The value encoded in __init__.py should not be the release version. The version will be updated as part of wheel builds to set the release. We'll probably want to institute something similar for conda builds, but things being out of sync in that respect is the status quo. There are other problems with using true alpha versions (our nightlies do not have such versions), and the version of a conda package as determined by the metadata on conda already doesn't match `__version__` in those packages, so I think all of these are future problems.